openmetadata-spec/src/main/resources/json/schema/entity/services/connections/database/trinoConnection.json

```diff
@@ -53,6 +53,11 @@
       "description": "Catalog of the data source.",
       "type": "string"
     },
+    "database": {
```

Comment:
Thanks for the help, just a couple of concerns here:
- For Presto and Trino we are using the Catalog field as the database in OM. I would not add the `database` field here as it can be confusing.
- For Glue, we use the catalog ID. Thus, same as above, I would not add the database field.
- Databricks same same, uses the catalog. So Let's not add the database field in there.

The idea here is to allow different names if, and only if, the user needs it. There is no concept of database as such in these connectors. I would do a couple of changes in the logic of this change:
- I'd name the field `databaseName`: We are not ingesting a different database. We'll be ingesting the schemas present in the source as databaseSchema and just change the database name. This database is just a filler to comply with OM's entity hierarchy.
- I'd have this field with a default value "default". This is the one we will use in the default implementation of [get_database_names](https://github.com/open-metadata/OpenMetadata/blob/main/ingestion/src/metadata/ingestion/source/database/common_db_source.py#L131).
- Let's remove the logic change in the connection builder. This change should be only aesthetic and have no impact on how the connection gets created. Even adding the check for "supportsDatabase", having the name as "database" in the JSON Schema can be confusing.

Thanks